web/src/main/webResources/WEB-INF/config-db/jdbc.properties

```diff
@@ -40,7 +40,7 @@ jdbc.basic.minEvictableIdleTimeMillis=1800000
 jdbc.basic.testWhileIdle=true
 jdbc.basic.numTestsPerEvictionRun=5
 jdbc.basic.poolPreparedStatements=true
-jdbc.basic.maxOpenPreparedStatements=1200
+jdbc.basic.maxOpenPreparedStatements=50
```

Comment:
This seems that was increased to solve this issue https://github.com/geonetwork/core-geonetwork/issues/521.

@fxprunayre do you think this is still an issue?

Reply:
Not sure, it is 10 years old. We updated drivers and hibernates since that time. Did you try batch delete ?